.github/dependabot.yml

```diff
@@ -10,6 +10,8 @@ updates:
   directory: "/"
   schedule:
     interval: monthly
+  ignore:
+    - dependency-name: "mockery"
```

Comment:
Please add a comment explaining why we ignore this.

Reply:
PTAL.